spec/features/user/user_can_edit_their_information_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe "User can update their information" do
+  scenario "User change their information" do
+    user = create(:user, first_name:"Jane", username: "jane", password: "password")
+
+    visit login_path
+
+    fill_in "Username", with: "jane"
+    fill_in "Password", with: "password"
+    click_on "Login"
+
+    expect(page).to have_content("Welcome, Jane!")
+    save_and_open_page
```

Comment:
Thank you for building the test for that @edilenedacruz , I don't know how we could have forgot to have this test to test that user can edit the profile.
All looks good, merging it shortly.